test-cases/gutenberg/reusable.md

```diff
@@ -0,0 +1,104 @@
+
+# Reusable Block - Test Cases
+
+## Preparation
+
+**For testing this block, it's required to create reusable blocks via the web version, here are the steps:**
+
+- Navigate to reusable blocks page (URL: `https://wordpress.com/types/wp_block/<YOUR_WPCOM_SITE_DOMAIN>`).
+- Click on "Add new reusable block".
+- Add a title.
+- Add some blocks.
+- Publish it.
+
+The reusable blocks page can be also used for editing and deleting blocks.
+
+##### TC001
+
+**Add block**
+
+**Steps**
+- Tap on ➕ button.
+- Tap on "Reusable" tab.
+- Tap on a reusable block.
+- Once inserted, tap on the block.
```

Comment:
Wdyt about adding a "to edit it" at the end of the sentence? Looks like that's how describe the goal [later in the text](https://github.com/wordpress-mobile/test-cases/pull/79/files#diff-6c976df25a3a675826c7dcc37f0922bd611004286dcba71e631192648fdd84c5R33).

Reply:
Yeah, it will be clearer to add that part as the action is meant to edit the block.

Reply:
I see this got addressed with https://github.com/wordpress-mobile/test-cases/pull/79/commits/d9411dd6cde52786887535b7104a17a6c3978d16#diff-6c976df25a3a675826c7dcc37f0922bd611004286dcba71e631192648fdd84c5R25, thanks!